ekino/newrelic-bundle/1.3/etc/packages/prod/ekino_new_relic.yaml

```diff
@@ -0,0 +1,17 @@
+ekino_new_relic:
```

Comment:
Naming the yaml file with the config root key could be considered as a best practice.